### PR TITLE
Translation 4 files to Ko

### PR DESCRIPTION
**Pull request by bumkeyy**
en : 
* [Command_line_Options_6171.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//tsconfig-reference/copy/en/categories/Command_line_Options_6171.md)
* [Experimental_Options_6177.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//tsconfig-reference/copy/en/categories/Experimental_Options_6177.md)
* [Module_Resolution_Options_6174.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/tsconfig-reference/copy/en/categories/Module_Resolution_Options_6174.md)
* [Project_Files_0.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//tsconfig-reference/copy/en/categories/Project_Files_0.md)

번역 리뷰 요청합니다 : @yeonjuan @guyeol @dvlprsh

ref #910

### CODE REVIEW

**Review by guyeol — APPROVED** [review]
LGTM!  Great works👍

File: packages/tsconfig-reference/copy/ko/categories/Experimental_Options_6177.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+display: "Experimental"
+---
+
+TypeScript는 JavaScript 언어에 추가 된 것으로 확인 된 기능만 포함하기 위해 노력합니다.
```
Comment (yeonjuan) [inline packages/tsconfig-reference/copy/ko/categories/Experimental_Options_6177.md]:
```suggestion
TypeScript는 JavaScript 언어에 확실히 추가될 기능만 포함하기 위해 노력합니다.
```

to be 는 미래를 의미하는 것 같아서 수정했습니다 :)

File: packages/tsconfig-reference/copy/ko/categories/Experimental_Options_6177.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+어떤 기능은 해당 규칙에 대한 예외가 될 만큼 강력한 경우가 있으며, 실험적인 컴파일러 플래그로 존재할 수 있습니다.
```
Comment (yeonjuan) [inline packages/tsconfig-reference/copy/ko/categories/Experimental_Options_6177.md]:
```suggestion
어떤 기능은 이 규칙에서 예외가 될 만큼 강력한 경우가 있으며, 실험적인 컴파일러 플래그로 존재합니다.
```

**Comment by bumkeyy** [comment]
@guyeol @yeonjuan  Thanks :)

**Comment by bumkeyy** [comment]
LGTM

**Comment by orta** [comment]
Oh interesting, all code owner checks are failing ATM - https://github.com/microsoft/TypeScript-Website/runs/1059756506?check_suite_focus=true

**Pull request merged**